mantle/cmd/plume/prerelease.go

```diff
@@ -580,46 +564,23 @@ func awsUploadAmiLists(ctx context.Context, bucket *storage.Bucket, spec *channe
 	jsonAll := jsonBuf.String()
 
 	// format text AMI lists and upload AMI IDs for individual regions
-	var hvmRecords, pvRecords []string
+	var records []string
 	for _, entry := range amis.Entries {
-		hvmRecords = append(hvmRecords,
-			fmt.Sprintf("%v=%v", entry.Region, entry.HvmAmi))
-		if entry.PvAmi != "" {
-			pvRecords = append(pvRecords,
-				fmt.Sprintf("%v=%v", entry.Region, entry.PvAmi))
-		}
+		records = append(records,
+			fmt.Sprintf("%v=%v", entry.Region, entry.Ami))
 
 		if err := upload(fmt.Sprintf("hvm_%v.txt", entry.Region),
-			entry.HvmAmi+"\n"); err != nil {
+			entry.Ami+"\n"); err != nil {
 			return err
 		}
-		if entry.PvAmi != "" {
-			if err := upload(fmt.Sprintf("pv_%v.txt", entry.Region),
-				entry.PvAmi+"\n"); err != nil {
-				return err
-			}
-			// compatibility
-			if err := upload(fmt.Sprintf("%v.txt", entry.Region),
-				entry.PvAmi+"\n"); err != nil {
-				return err
-			}
-		}
 	}
-	hvmAll := strings.Join(hvmRecords, "|") + "\n"
-	pvAll := strings.Join(pvRecords, "|") + "\n"
+	textAll := strings.Join(records, "|") + "\n"
 
 	// upload AMI lists
 	if err := upload("all.json", jsonAll); err != nil {
 		return err
 	}
-	if err := upload("hvm.txt", hvmAll); err != nil {
-		return err
-	}
-	if err := upload("pv.txt", pvAll); err != nil {
-		return err
-	}
-	// compatibility
-	if err := upload("all.txt", pvAll); err != nil {
+	if err := upload("hvm.txt", textAll); err != nil {
```

Comment:
Should we rename this to something like `all.txt`?

Reply:
Turns out this entire function is dead code.